Remove Studio quickstart content from local server quickstart (#206)
langchain-ai/docs: src/langgraph-platform/quick-start-studio.mdx

sidebarTitle: Use LangGraph Studio
---

[LangGraph Studio](/langgraph-platform/langgraph-studio) supports connecting to two types of graphs:

- Graphs deployed on [LangGraph Platform](#langgraph-platform).
- Graphs running locally via the [LangGraph Server](#local-development-server).

## LangGraph Platform

LangGraph Studio is accessed from the LangSmith UI, within the **LangGraph Platform Deployments** tab.

For applications that are [deployed](/langgraph-platform/deployment-quickstart) on LangGraph Platform, you can access Studio as part of that deployment. To do so, navigate to the deployment in LangGraph Platform within the LangSmith UI and click the **LangGraph Studio** button.

This will load the Studio UI connected to your live deployment, allowing you to create, read, and update the [threads](/oss/persistence#threads), [assistants](/langgraph-platform/assistants), and [memory](/oss/memory) in that deployment.

## Local development server

> To test your locally running application using LangGraph Studio, ensure you have set up your application following [this local application quickstart](/langgraph-platform/local-server).

Reviewer: ```suggestion
To test your application locally using LangGraph Studio, follow the [local application quickstart](/langgraph-platform/local-server) first.
```

<Info>
**LangSmith Tracing**

> For local development, if you do not wish to have data traced to LangSmith, set `LANGSMITH_TRACING=false` in your application's `.env` file. With tracing disabled, no data will leave your local server.

Reviewer: ```suggestion
For local development, if you don't want data traced to LangSmith, set `LANGSMITH_TRACING=false` in your application's `.env` file. With tracing disabled, no data leaves your local server.
```
